WORKSPACE

```diff
@@ -7,7 +7,7 @@ envoy_api_dependencies()
 load("//bazel:repositories.bzl", "GO_VERSION", "envoy_dependencies")
 load("//bazel:cc_configure.bzl", "cc_configure")
 
-envoy_dependencies()
```

Comment:
You probably didn't want this one. Also, format needs fixing.

Reply:
Thanks for catching that, should be good now.